test/forcing/OptionalWrappedProvider_Test.cpp

```diff
@@ -92,7 +92,7 @@ TEST_F(OptionalWrappedProvider_Test, test_is_default_override_1_b) {
 
     // Try again, after getting the (default) value once, meaning it should no longer override
     // Args don't really matter (apart from the name) for backing trivial item
-    double output_value = optProvider.get_value(OUTPUT_NAME_1, 0, 10, "m");
+    double output_value = optProvider.get_value(CatchmentAggrDataSelector("",OUTPUT_NAME_1, 0, 10, "m"), data_access::SUM);
```

Comment:
This class is only used in multi-BMI formulations (I think)...use `BMIDataSelector` in this test class maybe?